cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -171,6 +185,15 @@ private void reconcileAll(String trigger) {
         return result.future();
     }
 
+    private void setupMetrics() {
+        new ClassLoaderMetrics().bindTo(registry);
+        new JvmMemoryMetrics().bindTo(registry);
+        new ProcessorMetrics().bindTo(registry);
+        new JvmThreadMetrics().bindTo(registry);
+        new JvmGcMetrics().bindTo(registry);
+        new KafkaConsumerMetrics().bindTo(registry);
```

Comment:
I wonder if the `KafkaConsumerMetrics` have any value given the Cluster operator does not use any actual KafkaConsumer but only an KAfka Admin client. Does it return any metrics?

Reply:
I was thinking about this. I have removed the KafkaConsumer metrics.